WcaOnRails/app/models/round.rb

```diff
@@ -115,4 +121,35 @@ def self.wcif_json_schema
       },
     }
   end
+
+  # Compute a round type id from round information
+  # The round index is expected to **start from 1**
+  def self.compute_round_type_id(round_index, total_rounds, has_cutoff)
```

Comment:
@SAuroux I'd really like to have the WRT point of view on this function!
It's designed to figure out the round type from: the round number, the total number of rounds for the event, if it has cutoff.
I intentionally did not handle qualification rounds and such, as they are not used anymore.